src/locales/en.json

```diff
@@ -155,7 +155,7 @@
     },
     "saved_scenario_loaded_message": "You selected this scenario: <strong>{{ scenarioName }}</strong>. Do the points below look like the activity you have in mind? If not, adjust it until it looks right.",
     "alerts": {
-      "b117": "<0>Model update:</0> We've adjusted parameters for the new COVID-19 variant, B117. Read more <3>here.</3>",
+      "top_alert": "<0>Model update:</0> Our mask options have been expanded and updated. For more details, see the <3>White Paper.</3>",
```

Comment:
This key is confusing. I think it should be "masks_update" -- we want keys to describe the string not the location. That way, we create brand new keys when the content changes significantly, which triggers alerts to all the translators.

Reply:
oh!

Reply:
Resolved.